model/userModel.js

```diff
@@ -0,0 +1,19 @@
+const { DataTypes } = require('sequelize');
+const sequelize = require('../db/db'); // Import the Sequelize instance
+
+const User = sequelize.define('users', {
+  user_id: {
+    type: DataTypes.UUID, 
```

Comment:
```suggestion
    type: DataTypes.UUID.V4, 
```
Please check if this works before commit https://sequelize.org/docs/v7/models/data-types/#built-in-default-values-for-uuid